release-notes.md

```diff
@@ -1,5 +1,16 @@
 # Release notes
 
+## Workflows with Secure Device Access v1.1.0
+
+### What's new:
+- Added two-factor authentication (2FA) feature to the Android app.
+- Added a Captcha challenge when you enter a wrong password more than five times in the Android app.
+- Handled scenario in which the Android app went into **Pelion DM isn't responding** state for write tasks of more than 2kB.
```

Comment:
Does the app say "Pelion DM" or does it spell out "Device Management"? Our branded product name is "Pelion Device Management" on first use and "Device Management" on subsequent uses. This also applies to lines 11 and 22.

Reply:
On launcher and inside app drawer app says "Pelion DM". We have not spell out complete name at launcher because of space constraint but it implies Pelion Device Management.